docs/reference/release-notes/25.10-release-notes.md

```diff
@@ -0,0 +1,75 @@
+(reference-release-notes-25-10)=
+# 25.10 release notes
+
+> Landscape 25.10 runs on Ubuntu 24.04 LTS Noble Numbat or 22.04 LTS Jammy Jellyfish. Database schema changes are required to upgrade to Landscape Server 25.10.
+
+You can now access Landscape 25.10 in our `latest-stable` rolling release PPA: `ppa:landscape/latest-stable`.
+
+Note that 25.10 is not an LTS release. For self-hosted production deployments, we recommend using an LTS release, such as {ref}`Landscape Server 24.04 LTS <reference-release-notes-24-04-lts>`, which come with 10 years of support.
+
+## Highlights
+
+- **WSL profiles**: Install selected WSL instances on Landscape-registered Windows clients. Ensure that only WSL instances matching your profile are installed. Choose from Ubuntu LTS images or supply your own image. Customize installed WSL instances with [cloud-init](https://cloud-init.io/).
```

Comment:
```suggestion
- **WSL profiles**: Install selected WSL instances on Landscape-registered Windows clients. Ensure that only WSL instances matching your profile are installed. Choose from Ubuntu LTS images or supply your own image. Customize installed WSL instances with [cloud-init](https://cloud-init.io/). See {ref}`how-to-use-wsl-profiles` to learn more.
```
I forgot that we won't be able to link to the other docs... maybe in the future, the release notes and migrating docs from staging should be done in the same PR (I'll make a note to reconsider that)

When we're done with this PR, I can be in charge of merging it and publish the other docs first and test these links (if we include them)